lib/dot.js

```diff
@@ -12,6 +12,9 @@ export default class Dot extends TileLayer.Canvas {
   initialize (grid, color = '#49a0d7') {
     this.color = new Color(color).rgbArray()
     this.grid = grid
+    //this.colorScale = new CutoffColorScale(1000, [0, 0, 0, 0], [...this.color, 128])
+    this.colorScale = new InterpolatingColorScale([[200, 0, 0, 200, 0], [2000, 0, 0, 200, 180], [20000, 150, 0, 0, 180]])
```

Comment:
After my last push, it now handles rendering all grids, interpolated or not, with configurable color scales. So yes, it needs to be renamed and have a ColorScale parameter when constructed. I don't fully understand why there's an initialize() method instead of a constructor though.